wvd-sh/WVD scaling script/basicScale.ps1

```diff
@@ -260,12 +285,51 @@ if ($EndPeakDateTime -lt $BeginPeakDateTime) {
 }
 $hostpoolInfo = Get-RdsHostPool -TenantName $tenantName -Name $hostPoolName
 if ($hostpoolInfo.LoadBalancerType -eq "DepthFirst") {
+
 Write-Log 1 "$hostPoolName hostpool loadbalancer type is $($hostpoolInfo.LoadBalancerType)" "Info"
+
+    <#
+    if ($hostpoolMaxSessionLimit -eq 2) {
+      $sessionlimit = $hostpoolMaxSessionLimit - 1
+    }
+
+
+
+    else {
+      $sessionlimitofhost = $hostpoolMaxSessionLimit / 4
+      $var = $hostpoolMaxSessionLimit - $sessionlimitofhost
+      $sessionlimit = [math]::Round($var)
+    }
+
+    #>
+     $hostpoolMaxSessionLimit = $hostpoolinfo.MaxSessionLimit
+    if ($hostpoolMaxSessionLimit -le 10) {
+      $sessionlimit = $hostpoolMaxSessionLimit - 1
+
+    }
+
+
+    elseif ($hostpoolMaxSessionLimit -le 50) {
+      $sessionlimitofhost = $hostpoolMaxSessionLimit / 4
+      $var = $hostpoolMaxSessionLimit - $sessionlimitofhost
+      $sessionlimit = [math]::Round($var)
+
+
+	}
+    elseif ($hostpoolMaxSessionLimit -gt 50)
+    {
+      $sessionlimit = $hostpoolMaxSessionLimit - 10
+
+    }
+
+    Write-Log 1 "Hostpool Maximum Session Limit: $($hostpoolMaxSessionLimit)"
+
+  
   if ($CurrentDateTime -ge $BeginPeakDateTime -and $CurrentDateTime -le $EndPeakDateTime) {
 
-    Write-Log 1  "It is in peak hours now" "Info"
+    Write-Log 1 "It is in peak hours now" "Info"
     Write-Log 1 "Peak hours: starting session hosts as needed based on current workloads." "Info"
-    $hostpoolMaxSessionLimit = $hostpoolinfo.MaxSessionLimit
+    
     ##### Get the session hosts in the hostpool #####
     try {
       $getHosts = Get-RdsSessionHost -TenantName $tenantname -HostPoolName $hostpoolname | Sort-Object $_.sessionhostname
```

Comment:
try/catch: The only error here is that "HostPoolName" does not exist in the "TenantName". Ensure that you have entered the correct values.

Reply:
We have done changes